_posts/2020-05-01-status-update.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "01st May 2020: IAC Team Status Update"
+author: sheenaajay
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - community
+  - litmus
+  - testing
+  - dsc
+  - pdk
+  - powershell
+  - windows
+  - modulebuilder
+  - CampToCamp
+  - voxpupuli
+---
+
+The IAC team has been working on a couple of interesting things in the past week.
+Here's a brief overview of what we've been up to:
+
+## Virtual Puppet Camp Germany
+
+Puppet Camp Berlin has gone virtual.
+We invite you to join us for an immersive and educational event without the risk of travel.
+
+Join us online at [Virtual Puppet Camp Germany][Virtual Puppet Camp Germany] for technical talks from local leaders to level up your Puppet skill.
+
+- Date And Time
+Tue, May 5, 2020
+9:00 AM – 12:30 PM CEST
+
+[Paula Muir][pmcmaw] will be one of the MCs for the Puppet Camp.
+
+## Module Releases
+
+One thing we are always happy to talk about is the release of a module, in this case:
+
+- [puppetlabs/inifile](https://forge.puppet.com/puppetlabs/inifile) (v4.2.0)
+
+## puppet-modulebuilder Gem 0.2.0 Release
+
+[puppet-modulebuilder][puppet-modulebuilder] gem, which has now [been released](https://rubygems.org/gems/puppet-modulebuilder/versions/0.2.0)!
+
+Thanks a lot to [ekohl](ekohl) for the improvement on puppet-modulebuilder!
+
+## Continued Work on Puppet DSC
+
+The [Puppet DSC project][iac-41] continues on through phase II, with this week seeing the ability to Puppetize a PowerShell module's DSC resources with a single command (New-PuppetDscModule), as well as improving the output modules themselves, ensuring that their reference documentation and readme are generated.
+The other big change in the last week was to the underlying provider and type generation, improving debuggability and fully mapping embedded CIM instances!
+
+To any community members who would like to be part of our beta testing program, I encourage you to reach out to [dsc@puppet.com](mailto:dsc@puppet.com) with answers to the following questions:
+
+- What are the top three PowerShell modules with DSC resources you use/depend on the most?
+- Are you interested in participating in our beta testing program?
```

Comment:
Do we need to state this if the sentence above is asking if they are interested in being part of the beta testing program?

Reply:
@sanfrancrisko yeah make sense. I will remove the line. Thank you.